pkg/live/dual-delegating-provider_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
should the attribution here be to the Kubernetes Authors?  This is in kpt repo.

Reply:
Good catch. It should be: `// Copyright 2020 Google LLC`

Reply:
Fixed for all files in this PR. A small subsequent PR will fix others.